ui/src/app/core/initial-error/initial-error.component.html

```diff
@@ -10,16 +10,7 @@
       <tr>                                               |___/           </tr>
   </table>
   <p>
-  <p>
-    Job Manager is running but encountered a problem talking to its workflow server.
-  </p>
-  <p>
-    Please check the URL and workflow server configuration, or contact your support organization for assistance.
-  </p>
-  <p>
-    <a href="/">Click here to start over.</a>
-  </p>
-  <p>
-    <i>{{errorMessage}}</i>
-  </p>
+  <p>Job Manager is running but encountered a problem getting data from its workflow server.</p>
```

Comment:
I kind of agree that a way to make this more specific for 401s might be nice.

eg I could see this page being sort of contradictory - eg "is it broken...? or should I log back in...? argh!!"

Reply:
Ignore this - has now been changed.